built-in-references/KubernetesService/container-allowed-images/limited-preview/azurepolicy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "displayName": "[Limited Preview]: Ensure only allowed container images in AKS",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.ContainerService.Data",
+    "description": "This policy ensures only allowed container images are running in an Azure Kubernetes Service cluster. Limited Preview policies only work for registered subscriptions. To register, please go to https://aka.ms/akspolicyonboarding. For instruction on using this policy, please go to https://aka.ms/akspolicydoc.",
+    "metadata": {
+      "category": "Kubernetes service"
+    },
+    "parameters": {
+      "allowedContainerImagesRegex": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Allowed container images regex",
+          "description": "Regex representing container images allowed in Kubernetes cluster. E.g. Regex of azure container registry images is ^.+azurecr.io/.+$"
+        }
+      },
+      "effect": {
+        "type": "String",
+        "defaultValue": "EnforceRegoPolicy",
+        "allowedValues": [ "EnforceRegoPolicy", "Disabled" ],
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        }
+      }
+    },
+    "policyRule": {
+      "if": {
+        "field": "type",
+        "equals": "Microsoft.ContainerService/managedClusters"
+      },
+      "then": {
+        "effect": "[parameters('effect')]",
+        "details": {
+          "policyId": "ContainerAllowedImages",
+          "policy": "https://raw.githubusercontent.com/Azure/azure-policy/master/built-in-references/KubernetesService/container-allowed-images/limited-preview/gatekeeperpolicy.rego",
+          "policyParameters": {
+            "allowedContainerImagesRegex": "[parameters('allowedContainerImagesRegex')]"
+          }
+        }
+      }
+    }
+  },
+  "id": "/providers/Microsoft.Authorization/policyDefinitions/5f86cb6e-c4da-441b-807c-44bd0cc14e66",
+  "name": "5f86cb6e-c4da-441b-807c-44bd0cc14e66"
```

Comment:
should remove these to allow users to consume these files

Reply:
Liz told me to provide the built-in definitions here and I referred the other built-in samples https://github.com/Azure/azure-policy/blob/master/samples/built-in-policy/enforce-tag-value/azurepolicy.json, they are also in similar fashion. 

For creating new policies using these definitions, azurepolicy.parameters.json and azurepolicy.rules.json can be used.